### DISCUSSION TITLE
Sonoff devices not communicating after coordinator was offline

### DISCUSSION THREAD

**Discussion by jmachta** [Koenkk/zigbee2mqtt#19227] [Devices]
Hello all, 

I have a strange problem which I would like to discuss and ask if anybody has such experience too. 

I have quite a big zigbee network consisting of Aquara, Philips HUE, Ikea and Sonoff devices. This network runs basically 24/7 for more than year. However, due to my use case the coordinator gets once per several weeks offline for 2-3 days, occasionally longer. 

After booting up the system again, all devices are detected except those from Sonoff. These are mainly Sonoff SNZB-02 thermometers. To make these work, I need to remove them from network, power cycle them and bind them again. Otherwise they stay silent. Same happens with Sonoff SNZB-01 button.

Can anyone advise? Alternatively, is there any other device that works in such use case correctly and costs about the same as Sonoff ones?

It is maybe worth mentioning that I am using Sonoff Zigbee 3.0 USB Dongle Plus.

**Comment by Lollos60** [comment]
Same problem here with Legrand-Bticino devices and Sonoff dongle P (20221226 Z-Stack firmware)
